js/plugin/Routing.js

```diff
@@ -252,7 +252,9 @@ BR.Routing = L.Routing.extend({
     // change segment color before request to indicate recalculation (mark old)
     if (m1 && m1._routing.nextLine !== null) {
         m1._routing.nextLine.options.color = 'dimgray';
-        m1._routing.nextLine._updateStyle();
+
+        console.log("TODO: fixme! //m1._routing.nextLine._updateStyle()");
+        // m1._routing.nextLine._updateStyle();
```

Comment:
The fix in commit [Use setStyle instead of _updateStyle](https://github.com/bagage/brouter-web/commit/54be34aa4ee22f1b85b18a19cfb6c2b175c86c67) doesn't seem to have been merged into the PR branch?